Makefile.include

```diff
@@ -129,6 +129,9 @@ ifeq (,$(UNZIP_HERE))
   endif
 endif
 
+ifeq (, ${APPLICATION})
+    $(error An application name must be specified as APPLICATION.)
```

Comment:
:-1: there is already to much assumption going around that this is always the case... (just a feeling, not rational thought)